pkg/asset/asset.go

```diff
@@ -36,9 +36,11 @@ const (
 // AssetConfig holds all configuration needed when generating
 // the default set of assets.
 type Config struct {
-	EtcdServers []*url.URL
-	APIServers  []*url.URL
-	AltNames    *tlsutil.AltNames
+	EtcdServers    []*url.URL
+	APIServers     []*url.URL
+	CAFile         string
+	PrivateKeyFile string
```

Comment:
I'm thinking we might want to require these to be x509 objects. This way we're not assuming how we retrieve this data (e.g. from disk), just that it is provided. In the case of other tools using this, we don't know that this data is going to be provided as a path.